src/airport.js

```diff
@@ -1 +1,91 @@
+class Airport {
+
+    constructor() {
+
+        this.planesInAirport = [];
+        this.airportCapacity = 10; // default 
+
+    }
+
```

Comment:
Should have created a separate class for 'planes' as the planes only exist as strings in the airport array. There is no concept of planes in the air, and they are deleted once the 'takeOff' method is used.

If I had a plane class, I could check the planes to see if they are at an airport or if they are in the air and I could land the planes at different airports.